### ISSUE TITLE
Expected linebreaks to be 'LF' but found 'CRLF' linebreak-style

### ISSUE THREAD

**Issue by dlemstra** [twilio-labs/twilio-style#24]
When I tried to use this configuration in the `create-twilio-function` project I got spammed with the following error:

`Expected linebreaks to be 'LF' but found 'CRLF' linebreak-style`

This is happening because I am working on a Windows machine. This can be resolved by either removing this line:

https://github.com/twilio-labs/twilio-style/blob/37a78d0b000a40632853125403f75c94575b6e8b/packages/eslint-config-twilio/rules/stylistic-issues.js#L71

or by adding a .gitattributes file to all the projects that use this config.

**Comment by ahcai** [comment]
@dlemstra You can add an override in your repo to turn that rule off in your `.eslintrc`.
```
{
  "extends": [
    "twilio"
  ],
  "rules": {
    "linebreak-style": "off"
  }
}
```

**Comment by dlemstra** [comment]
Could you clarify why I should turn that rule off in a repository @ahcai? This would mean that this would needs to be turned off in any repository that also builds on Windows and has no `.gitattributes` file forcing `LF`. Are there plans to add `.gitattributes` files to all the repositories? And then turn that rule back on?

**Comment by ktalebian** [comment]
We cannot only set the linebreak to be either LF (for Unix) or CRLF (for Windows) but not both at the same time, so whichever rule we choose to, is going to upset some developers.

We chose Unix because that's recommended [ESLint rule](https://eslint.org/docs/rules/linebreak-style) as well as the [AirBnB rule](https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/style.js#L179).

**Comment by dlemstra** [comment]
Should we not add a `.gitattributes` file to the repository to force `LF` in the repositories that use this library?

**Comment by ahcai** [comment]
@dlemstra We'll do that!

**Comment by ahcai** [comment]
This is fixed in v1.15.2.
